app/services/sirsi_response/error.rb

```diff
@@ -9,6 +9,8 @@ class SirsiResponse::Error
     "hatErrorResponse.105": 'Denied: Item has been recalled, cannot be renewed.',
     "hatErrorResponse.252": 'Denied: Item has holds, cannot be renewed.',
     "hatErrorResponse.46": 'Denied: Item on reserve, cannot be renewed.',
+    "hatErrorResponse.238": 'This item cannot be automatically renewed. Please contact '\
+          '<a href="mailto:ul-lending@lists.psu.edu">ul-lending@lists.psu.edu</a> for assistance in renewing the item.',
```

Comment:
```suggestion
          '<a href="mailto:ul-lending@lists.psu.edu">ul-lending@lists.psu.edu</a> for assistance in renewing the item.'.html_safe,
```

This is actually what I was thinking, but, does that not work how I am thinking it does?

Reply:
Separately I wonder why we have double quotes for the keys of this `Hash`. Seems kinda weird considering we never use double quotes for strings without interpolation anywhere else and rubocop usually yells at us for that. 🤷

Reply:
Could also use [`mail_to`](https://apidock.com/rails/v5.2.3/ActionView/Helpers/UrlHelper/mail_to), instead of marking the html as safe, like:

```suggestion
          "mail_to('ul-lending@lists.psu.edu') for assistance in renewing the item.",
```

Reply:
ah this is even better 👍  thanks

Reply:
> Separately I wonder why we have double quotes for the keys of this `Hash`. Seems kinda weird considering we never use double quotes for strings without interpolation anywhere else and rubocop usually yells at us for that. 🤷

yes very weirdly rubocop does not yell at this. I will change those quotes

Reply:
just tried mail_to but did not work. I think it requires helper method to be included in this model.

Reply:
ah sorry "#{mail  to here}"

Reply:
I'd tried that too :/

Reply:
just adding a note here about convo in slack: basically sidekiq loaded string here does not have the method html_safe and the mail_to helper is undefined it seems which seems to imply that RoR isn't properly loaded/initialized for sidekiq jobs perhaps 🤔